test/functional/test_site/testNunjuckUtils.md

```diff
@@ -0,0 +1,9 @@
+{%raw%}
+
+The standard trick to render braces still works {<span>{ content }</span>}.
+
+Nonexistent variables should still be removed: {{ unknown_variable }}. In the rare event that user wrapped an unknown variable inside a raw tag, it should retain its current behaviour and not be produced to ensure Vue does not break.
```

Comment:
Shouldn't it be un-removed? ( isn't this PR to let `{% raw %} ... {% endraw %}` work across multiple nunjucks renders? )

Reply:
The scope of this PR is only to escape nunjucks for special tags. This PR currently does not help to render `{{ unknown_variable }}` if it is not wrapped in a special tag. That is currently the purpose of `nunjucks.removeNunjucksEscapes`, as we still have to ensure that compatibility is maintained until the following is dealt with:

1. markdown-it-attr which also uses `{`, `}`. `md.render` would actually remove all the raw tags and add it as a HTML attribute.
1. We still need to escape `{{` and `}}` in Vue. A special component mentioned that uses `v-pre` might help.

Reply:
Hmm, I still don't follow 😅 
When we use your version of `nunjucks.removeNunjucksEscapes` for _all calls except the last_, shouldn't `{{ xxx }}` be preserved no matter?

> We still need to escape {{ and }} in Vue. A special component mentioned that uses v-pre might help.

> For general use, we could create a special component, something like <escape></escape> that acts the same as a <span v-pre></span> in addition to appending {% raw %} {% endraw %}

When I mentioned this, it's only a passing remark for a future PR, such that `{{ xxx }}` would be skipped in vue compilation and show up in the browser.

Before that though, shouldn't `{{ xxx }}` show up in the html output though? ( `markdown-it-attrs` does not erase `{{ xxx }}`, nor `{ xx }` if it is not placed appropriately. https://npm.runkit.com/markdown-it-attrs )

Reply:
> We would still require this. As explained above, we still need to ensure {% raw %} tags does not change current behavior by interfering with markdown-it-attr, now that it can be "retained" until the parser stage.

Ahh my bad, I see now that `{% raw %}` is the offender, not `{{ xxx }}`

Reply:
Yea :sweat:. 

For that, since it is not possible to change delimiters for both nunjucks and markdown-it-attr, I am separately testing whether it is possible to ignore nunjuck raw tags for markdown-it-attr: https://github.com/arve0/markdown-it-attrs/blob/a40f59a4ee690e2464c553b14a2e57e48d5418ae/utils.js#L101